modules/module-extras.php

```diff
@@ -23,6 +23,7 @@
 	'seo-tools/jetpack-seo-titles.php',
 	'seo-tools/jetpack-seo-posts.php',
 	'simple-payments/simple-payments.php',
+	'memberships/class-jetpack-memberships.php',
```

Comment:
Since the tool requires a WordPress.com connection, maybe we could move it with the SEO tools a bit downer in that file?

Reply:
I think it's working in the opposite way.
The modules right here are the ones that work only if JP is active,

The seo tools down there are working only if JP is INactive:

```
// Not every tool needs to be included if Jetpack is inactive and not in development mode
if ( ! Jetpack::is_active() && ! Jetpack::is_development_mode() ) {
	$tools = array(
		'seo-tools/jetpack-seo-utils.php',
		'seo-tools/jetpack-seo-titles.php',
		'seo-tools/jetpack-seo-posts.php',
	);
}
```

Reply:
Oh yeah you're right, that's confusing. Right now the default list is loaded as soon as the plugin is active, whether it's connected to WordPress.com or not, and includes things that need a connection as well as things that don't. It's not ideal. We'll need to refactor that in the future I think, to better separate the 2 scenarios. That can be done in a different PR though. 👍 	

Until then, I think it may be good to short-circuit early in `memberships/class-jetpack-memberships.php` if Jetpack is not connected to WordPress.com.

Reply:
Should that clarification be code-commented above the list here?

Reply:
I've just pushed #11796 to start reorganizing things a bit.